battle-engine.js

```diff
@@ -2045,6 +1955,8 @@ Battle = (() => {
 		this.messageLog = [];
 
 		this.startingSeed = this.generateSeed();
+
+		this.send = send;
```

Comment:
I would at least like `send` to be an optional argument, so you wouldn't need to mess with tests.